statistician/Dockerfile

```diff
@@ -36,3 +36,4 @@ COPY --from=env_builder $py_env_path $py_env_path
 
 RUN env && echo $PATH && pip freeze && pip check
 RUN odc-stats --help
+WORKDIR /tmp
```

Comment:
Let's do this earlier, please. The `RUN odc-stats --help` should go last as a smoke test.

Reply:
Done.